openfisca_core/entities.py

```diff
@@ -53,6 +53,23 @@ def __call__(self, variable_name, period = None, options = [], **parameters):
         else:
             return self.simulation.calculate(variable_name, period, **parameters)
 
+    # JSON
+
+    @classmethod
+    def to_json(cls):
+        entity_dict = {
+            'key': cls.key,
+            'label': cls.label,
+            'plural': cls.plural,
+            }
+        if cls.is_person:
+            entity_dict['isPersonsEntity'] = cls.is_person
```

Comment:
This is a common pattern, for `to_json` methods, to export only the fields which make sense. As a consequence, they avoid `key: null` or, in this case, `key: false`.

I agree that it would be better to export a `type` key whose value would be either `person` either `group`.

I kept the Web API function as is (see [here](https://github.com/openfisca/openfisca-web-api/blob/d790243123c72fd198cc29a63d6dd4dedb316510/openfisca_web_api/controllers/entities.py#L12-L20)) but I'm OK to enhance it *via* this PR.